### DISCUSSION TITLE
How can I hide the icons of taken photos on the map?

### DISCUSSION THREAD

**Discussion by bege10** [osmandapp/OsmAnd#21542] [Q&A]
I have taken photos with the notes plugin. I don't want their icons to always be displayed on the map. How can I hide them?

**Comment by dmpr0** [comment]
Hi! You can hide them by disabling **Recording layer** in **Menu** – **Configure Map**.

**Reply by bege10** [reply]
> Hi! You can hide them by disabling **Recording layer** in **Menu** – **Configure Map**.

Is "Recording layer" in the German locale "Tracks"? This does not hide the icons.
